tgui/packages/tgui/interfaces/DeathConsequencesPage.tsx

```diff
@@ -0,0 +1,85 @@
+import { useBackend } from '../backend';
+import { Button, NumberInput, Section, Stack } from '../components';
+import { PreferencesMenuData } from './PreferencesMenu/data';
+
+export const DeathConsequencesPage = (props, context) => {
+  const { data } = useBackend<DeathConsequenceData>(context);
+  return (
+    <Stack>
+      <Stack.Item maxWidth="99%">
+        <Section title="Death consequences config">
+          <Stack vertical pt="20px" wrap="wrap">
+            {data.dc_preferences.map((val) => (
+              <ConfigEntry key={val.typepath} preference={val} />
+            ))}
+          </Stack>
+        </Section>
+      </Stack.Item>
+    </Stack>
+  );
+};
+
+export const ConfigEntry = (props, context) => {
+  const { act } = useBackend<PreferencesMenuData>(context);
+  return (
+    <Stack.Item>
+      <Section title={props.preference.name}>
+        {props.preference.desc}
+        <br />
+
+        <Stack>
+          <Stack.Item>
+            {props.preference.interface_type === 'boolean' && (
+              <Button.Checkbox
+                name="ad"
```

Comment:
ad?

Reply:
i dont actually remember writing this, odd

regardless i need to touch this up before im done since preference lists can now easily be manually rendered